prime-router/metadata/schemas/WATERS/waters-covid-19.schema

```diff
@@ -247,6 +247,12 @@ elements:
     cardinality: ZERO_OR_ONE
     csvFields: [{ name: performingFacility}]
 
+  # Copy from testing_lab_clia
```

Comment:
I added reporting_facility_clia, because it looked like some states are grabbing testing_lab_clia and some are grabbing reporting_facility_clia.    We should probably get rid of one of these and consolidate, at least for covid-19.   It looked to me like all our incoming/sender schemas just copy the value into both fields.